experimental/https_attestation/Cargo.toml

```diff
@@ -36,4 +37,4 @@ tokio = { version = "*", features = [
   "sync",
   "rt-multi-thread"
 ] }
-tokio-rustls = "*"
+tokio-rustls = "0.22"
```

Comment:
Does the `# TODO(#2544)` apply here too?

Reply:
May or may not, I hope it will be more obvious when we do the update.